RELEASE_NOTES.md

```diff
@@ -19,8 +19,7 @@ More information on the contribution process for Chef projects can be found in t
 https://github.com/chef-partners/azure-chef-extension
 
 
-##Feature added in Windows azure-chef-extension 1210.12.109.1000
-* Add support for environment variables in windows chef extension [Issue 193](https://github.com/chef-partners/azure-chef-extension/pull/193)
-
-##Known issue
-* Extension works fine for the latest windows 2008 images. 
\ No newline at end of file
+## Feature added in Windows azure-chef-extension 1210.12.109.1002

```

Comment:
Also need to add https://github.com/chef-partners/azure-chef-extension/pull/203 too